src/blog/2022/08/flowforge-08-released.md

```diff
@@ -0,0 +1,70 @@
+---
+title: FlowForge 0.8 released
+subtitle: Inter-Project Communication, Default Teams, and realtime device management.
+description: Inter-Project Communication, Default Teams, and realtime device management.
+date: 2022-08-04 12:00:00.0
+authors: ["sam-machin"]
+video: BBNzFInc_2E
+---
+TODO! Update Video ID<br>
+
+Easily pass messages between your projects on the cloud or devices, UX improvements, and more.
+
+<!--more-->
+
+Keep reading for  the details of whats in this release our you can watch our 1 min roundup video of the new release above. 
+
+We're pleased to announce version 0.8 is now available. the next release of the FlowForge application.
+
+## Features
+[Project Link Nodes](https://github.com/flowforge/flowforge/issues/662)
+We've introduced some custom FlowForge nodes to the pallette of new projects, These are Project Link nodes that allow you to easily pass data between different projects within the same team. 
```

Comment:
```suggestion
We've introduced some custom FlowForge nodes to the palette of new projects. These are Project Link nodes that allow you to easily pass data between different projects within the same team. 
```

Reply:
Hmm wonder why my spell checker missed that